### DISCUSSION TITLE
Update relative to support for devices without Google Play Service ?

### DISCUSSION THREAD

**Discussion by BenStirrup** [Agontuk/react-native-geolocation-service#277] [General]
Hello @Agontuk and thank you for your gigantic contribution to the React Native community through this lib :) .

I read that you recently updated the changelog https://github.com/Agontuk/react-native-geolocation-service/commit/2ab0b25f17bc31fc84a652aa9f4c6e69fc518e8b . Should I understand that Android devices without Google Play Service (i.e. recent Huawei phones for example) will be able to get their geoposition using a fallback implementation ?

Do you have an estimated time of delivery for a stable release including this fix ?

Thanks again.

**Comment by Agontuk** [comment]
It's already available in version `5.3.0-beta.1` and should be stable for use.
